ivy/functional/backends/torch/gradients.py

```diff
@@ -31,16 +30,25 @@ def execute_with_gradients(func, xs, retain_grads=False):
         y = func_ret
         rest = tuple()
     y = ivy.to_native(y)
-    x_grads_flat = list(
-        torch.autograd.grad(
-            [y],
-            [v for k, v in xs.to_iterator()],
+    if isinstance(y,ivy.Container):
```

Comment:
why is this check only added for PyTorch and not TensorFlow or JAX, do these `execute_with_gradients ` implementations already support `ivy.Container` instance for the return `y`?

Reply:
The Hypothesis tests only falsified on PyTorch so I missed to consider TF and JAX, and you are definitely right. I will double check TF and JAX